dist/yaml/ovnkube-master.yaml

```diff
@@ -25,6 +25,7 @@ spec:
         component: network
         type: infra
         openshift.io/component: network
+        beta.kubernetes.io/os: "linux"
```

Comment:
Actually we do not need the label here. We just need it on the nodeSelector, though this is harmless.